src/main/webapp/styles/action.css

```diff
@@ -18,4 +18,19 @@
 
 .card-image {
     width: 20em;
+}
+
+.page-header {
+    /* margin: 0 10em 0 10em !important; */
+    background: linear-gradient(
+        to bottom, 
+        var(--dark-main-1),
+        10%,
+        var(--light-bg-1)
```

Comment:
are the two pages supposed to have different color gradients?

Reply:
yeah but I can change that if we want

Reply:
hmm we can ask Maxwell for his opinion too, but I think I like the Action page color more than the About Us page

Reply:
I like the About Us Page color more in general, but I think the Action Page color would fit better for our site

Reply:
Yup same, the About Us color's definitely prettier, but the green Action color fits better.

Reply:
I'm not entirely sure how I feel about the gradient though? We only have solid colors everywhere else, so it feels a bit weird. Definitely open for debate though

Reply:
oh no problem I'll change both of them to green! and yeah I wasn't sure about the gradient- I was trying to get it to look more uniform with the homepage but I can take that out too

Reply:
fixed!